install_debs_into_image.sh

```diff
@@ -165,6 +165,16 @@ if [ "$(/bin/ls "$BAKERYDIR/debs-to-install/"*.deb 2>/dev/null)" ] ; then
 	chroot "$IMAGEDIR" sh -c "dpkg -i /tmp/debs-to-install/*.deb"
 fi
 
+# install local files
+LIST_INS="$BAKERYDIR/files-to-install/list.ins"
```

Comment:
I'm not sure about this change. Also there is not description about this feature.

Reply:
I would think this is more temporary than other changes, as normally files will be delivered only with debian packages.